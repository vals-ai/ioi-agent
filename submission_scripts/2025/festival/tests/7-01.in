version https://git-lfs.github.com/spec/v1
oid sha256:fcc766479047954c77608b6324de0223dae4fd82dfd9d6b535dbb5021544e07f
size 2200860
