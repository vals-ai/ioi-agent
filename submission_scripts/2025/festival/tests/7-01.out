version https://git-lfs.github.com/spec/v1
oid sha256:2590bfcee0e87869c68eec9a9797cdb28914404afa2209f2f318a4464e82ab66
size 644855
